test/quoteSpec.js

```diff
@@ -282,17 +282,15 @@ describe('Quotes', function () {
     })
   })
 
-  describe('quotes a multi-hop route', function () {
+  describe('if route has no curve, quotes a multi-hop route', function () {
     beforeEach(function * () {
-      const curve = new LiquidityCurve([ [0, 0], [10000, 20000] ]).toBuffer()
       this.config.routeBroadcastEnabled = false
       yield this.messageRouter.receiveRoutes({
         new_routes: [{
           source_ledger: 'eur-ledger.',
           destination_ledger: 'random-ledger.',
           min_message_window: 1,
-          source_account: 'eur-ledger.mary',
-          points: curve.toString('base64')
```

Comment:
yes, I created https://github.com/interledgerjs/five-bells-shared/commit/ab1ecb6b7688 for that, will continue working on it tomorrow.